doc/releases/release-notes-4.0.rst

```diff
@@ -465,17 +480,16 @@ Drivers and Sensors
 
   * Added initial support for Renesas RA8 AGT counter driver (:dtcompatible:`renesas,ra-agt`)
   * Added driver for Analog Devices MAX32 SoC series (:dtcompatible:`adi,max32-counter`).
+  * Updated the NXP counter_mcux_lptmr driver to support multiple instances of the lptmr
+    peripheral.
 
 * Crypto
 
   * Added support for STM32L4 AES.
-  * Deprecated the TinyCrypt shim driver ``CONFIG_CRYPTO_TINYCRYPT_SHIM``.
 
 * DAC
 
   * DAC API now supports specifying channel path as internal. Support has been added in STM32 drivers.
-  * Updated the NXP counter_mcux_lptmr driver to support multiple instances of the lptmr
-    peripheral.
   * Updated the initialization of clocks for the NXP LPTMR driver
```

Comment:
These lines (until 496) also need to be moved to the `counter` section- also:
- the NXP software watchdog timer seems to be a watchdog driver, not a counter or DAC driver. So the note about that should probably go into the watchdog section
- the `nxp,s32-sys-timer` is a timer driver, it should be in the "timer" section

Reply:
Thanks for catching this. PR https://github.com/zephyrproject-rtos/zephyr/pull/81472